src/dbnode/server/server.go

```diff
@@ -1281,6 +1280,8 @@ func withEncodingAndPoolingOptions(
 	aggregateQueryResultsPool := index.NewAggregateResultsPool(
 		poolOptions(policy.IndexResultsPool, scope.SubScope("index-aggregate-results-pool")))
 
+	// Set index options.
+	opts = opts.SetTruncateType(cfg.Transforms.TruncateBy)
```

Comment:
nit: Maybe move this above the `// Set index options` since this transforms values? i.e. maybe just comment with `// Set value transformations`?